src/main/java/domain/winningStatistics/PrizeMoney.java

```diff
@@ -0,0 +1,41 @@
+package domain.winningStatistics;
+
+import java.util.Arrays;
+
+public enum PrizeMoney {
+    THREE(3, false, 5000),
+    FOUR(4, false, 50000),
+    FIVE(5, false, 1500000),
+    FIVE_BONUS(5, true, 30000000),
+    SIX(6, false, 2000000000);
+
+    private final int matchingCount;
+    private final boolean isHavingBonusNumber;
+    private final int prizeMoney;
+
+    PrizeMoney(int matchingCount, boolean isHavingBonusNumber, int prizeMoney) {
+        this.matchingCount = matchingCount;
+        this.isHavingBonusNumber = isHavingBonusNumber;
+        this.prizeMoney = prizeMoney;
+    }
+
+    public static float calculateEarningsRate(WinningStatistics winningStatistics, int purchasedCount) {
+        int totalPrizeMoney = Arrays.stream(values())
+            .mapToInt(prizeMoney -> winningStatistics.get(prizeMoney) * prizeMoney.getPrizeMoney())
```

Comment:
굳이 getPrizeMoney를 하지 않아도 될 것 같은데 Enum의 이름과 필드의 이름이 동일해서 발생한 문제 같네요.
enum의 이름이 변경된다면 다음과 같이 사용해도 될 것 같아요.
```suggestion
            .mapToInt(바꾼이름 -> winningStatistics.get(바꾼이름) * prizeMoney)
```